zerver/openapi/zulip.yaml

```diff
@@ -4841,6 +4841,19 @@ paths:
 
                                         **Changes**: Before Zulip 6.0 (feature level 138), no limit was
                                         represented using the special value `0`.
+                                    moderation_request_channel_id:
+                                      type: integer
+                                      description: |
+                                        The ID of the private channel to which messages flagged for moderation
+                                        requests by users are sent. Moderators can use this channel to review and
+                                        act on reported content.
+
+                                        Will be `-1` if moderation requests are disabled.
+
+                                        Since these automated messages are sent by the server, this field is
+                                        primarily relevant to clients containing UI for changing it.
```

Comment:
I think this is wrong -- it should probably say "Clients should use whether moderation requests are disabled to determine whether to present a report message feature in their UI within a given organization."

Reply:
hmm did you mean "Clients should **check** whether moderation requests are disabled to ..." or am I misunderstanding the message?

Reply:
I've updated this for now,:
```
Clients should check whether moderation requests are disabled to
determine whether to present a report message feature in their UI
within a given organization.
```
Let me know if this matches with what you had in mind!